app/lib/methods/loadMessagesForRoom.ts

```diff
@@ -5,32 +5,41 @@ import log from '../../utils/log';
 import { getMessageById } from '../database/services/Message';
 import { generateLoadMoreId } from '../utils';
 import updateMessages from './updateMessages';
+import { IMessage, IRoom, SubscriptionType, TMessageModel } from '../../definitions';
+import sdk from '../rocketchat/services/sdk';
+import RocketChat from '../rocketchat';
 
 const COUNT = 50;
 
-async function load({ rid: roomId, latest, t }) {
-	let params = { roomId, count: COUNT };
+async function load({ rid: roomId, latest, t }: Pick<IRoom, 'rid' | 'latest' | 't'>) {
```

Comment:
We should type this as basic types for simplicity.
We need to work on those broader types.

For example, you said IRoom, but shouldn't it be ISubscription?
SubscriptionType is also broken, so use string for now.